docs/organizations/integrations/slack-integration.md

```diff
@@ -0,0 +1,25 @@
+# Organization Slack integration for Security issues
+
+This integration sends notifications to a Slack channel of your choice whenever Codacy detects new critical Security issues in any repository within your organization.
+
+## Installing the Slack integration
+
+To install the Slack integration:
+
+1.  Follow the [instructions on the Slack documentation pages](https://api.slack.com/messaging/webhooks) to add an Incoming Webhook to your Slack workspace and set it to post to a channel of your choice, and then copy the webhook URL.
+
+    The Incoming Webhook URL starts with `https://hooks.slack.com/services/`.
+
+1.  Open your organization **Integrations**, page **Slack**, paste the Incoming Webhook URL in the field, and click **Install Slack**.
```

Comment:
Same here, we could place the click action on a different step.

```suggestion
1.  Open your organization **Integrations**, page **Slack** and paste the Incoming Webhook URL in the field.
1.  Click **Install Slack**.
```

Reply:
This one would probably be too granular. We're already using the "and click X" formula in [multiple places](https://github.com/search?q=repo%3Acodacy%2Fdocs%20%22and%20click%22&type=code).

Reply:
I agree with having the "click X" together with one previous (simple) step.
But in this case, we have two other steps before the click: open and paste. That was the only reason I suggested to separate. 🤓 
Anyway, we can keep it as is as discuss this detail later on. 👍